CMakeLists.txt

```diff
@@ -1,16 +1,41 @@
-project(libhausbus)
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+
+option(LIBHAUSBUS_CMAKE_DEBUG "Set to on for some Debug output" Off)
+
+set(LIBHAUSBUS_VERSION_MAJOR 1)
+set(LIBHAUSBUS_VERSION_MINOR 0)
+set(LIBHAUSBUS_VERSION_PATCH 0)
```

Comment:
Am a bit puzzled: You mean minor 1 and major 0? No it is not used atm, but plan to implement config stuff so that find_package can be sued with CONFIG option.